### PR TITLE
Contact page WIP

### PR DESCRIPTION
**Pull request by SolivanLau**
Hello! updates for contact form:

Here is what I have worked on:
- contact-form now has a php file
- moved over contact-form logic out of **main.js** into its own file
- html contact form has **default behavior again**

@maxharrisnet can you please tinker around with the php and js?  in code, you are cc'd to receive emails as a test. I am new to php and am still learning to host the file on server side to see if it works

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -86,7 +86,13 @@
 		<!-- Contact Content -->
 		<main>
 			<!-- Contact Form -->
-			<form action="#" id="contact-form" class="contact-form">
+			<form
```
Comment (SolivanLau) [inline contact.html]: default behavior enabled again.

File: data/contact-form.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<?php
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+
+    // Retrieve form data from the $_POST array
+    $name = $_POST['name-input'];
+    $email = $_POST['email-input'];
+    $message = $_POST['message-input'];
+
```
Comment (SolivanLau) [inline data/contact-form.php:9]: simplified the form data to design. sending to our work emails in the meantime

File: data/contact-form.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    // Send email and handle success or failure
+    if (mail($to, $subject, $body, $headers)) {
+        header('Location: ' . $successURL);
+        exit;
+    } else {
+        header('Location: ' . $failURL);
+        exit;
+    }
+}
```
Comment (SolivanLau) [inline data/contact-form.php:42]: I have the header updating accordingly. Depending on boolean state, js will act accordingly

**Review by maxharrisnet — APPROVED** [review]
Looks good enough to test 🍨

**Pull request merged**